tests/integration/elder.test.ts

```diff
@@ -0,0 +1,79 @@
+import Elder, {
+  Model,
+  StringType,
+  type,
+  Adapter,
+  PostgresAdapter,
+  Collection
+} from '../../src'
+
+const config = { adapters: { default: { database: 'ash' } } }
+
+describe('Elder', () => {
+  test('model setup', async () => {
```

Comment:
So its more like "Creating a model" type thing. How would you name/group files in the integration folder?